misc/imagestream.yml

```diff
@@ -0,0 +1,5 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: training
```

Comment:
Since this is a image-classification pipeline I think we can give a more descriptive name here.